StreamChatClient_v3/Database/DatabaseSession_Tests.swift

```diff
@@ -24,14 +24,33 @@ final class DatabaseSession_Tests: XCTestCase {
                                         channel: channelPayload.channel)
         
         // Save the event payload to DB
-        database.write { (session) in
+        database.write { session in
             try session.saveEvent(payload: eventPayload)
         }
         
         // Try to load the saved channel from DB
         var loadedChannel: ChannelModel<DefaultDataTypes>? {
             database.viewContext.loadChannel(cid: channelId)
         }
+        
         AssertAsync.willBeEqual(loadedChannel?.cid, channelId)
+        
+        // Try to load the saved current user from DB
```

Comment:
This is not the "current" user. It's just "any" user. `CurentUser` model was introduced in #349.